### PR TITLE
Added some scrolling shortcuts

### PR DESCRIPTION
**Pull request by iainhouston**

### CODE REVIEW

File: source/reference/keyboard_shortcuts_osx.rst [hunk 1]
```diff
@@ -110,6 +110,21 @@ Find/Replace
 | ⌘ + ⇧ + F       | Find in files                                             |
 +-----------------+-----------------------------------------------------------+
 
+Scrolling
+------------------------
+
++-----------------+-----------------------------------------------------------+
+| Keypress        | Command                                                   |
++=================+===========================================================+
+| ⌃ + V           | Scroll down one window                                    |
```
Comment (FichteFoll) [inline source/reference/keyboard_shortcuts_osx.rst]: The terminology of a "window" is inappropriate here. Instead, use "view" or just "page". For ^L use just view.

Also, is there no binding for going upward?

File: source/reference/keyboard_shortcuts_osx.rst [hunk 2]
```diff
@@ -110,6 +110,21 @@ Find/Replace
...
++-----------------+-----------------------------------------------------------+
+| ⌃ + L           | Centre current line vertically in window                  |
```
Comment (FichteFoll) [inline source/reference/keyboard_shortcuts_osx.rst]: @guillermooo, do we have guidelines for american vs british english?

Comment (guillermooo) [inline source/reference/keyboard_shortcuts_osx.rst]: I've used American spellings from the beginning, so let's stick to that.

Comment (FichteFoll) [inline source/reference/keyboard_shortcuts_osx.rst]: Okay, please change this to `center` then, @iainhouston.

**Comment by FichteFoll** [comment]
Thanks.

**Pull request merged**